build/rpmbuild-docker/Dockerfile

```diff
@@ -19,8 +19,12 @@ RUN yum install -y --setopt=skip_missing_names_on_install=False \
 RUN yum -y remove nodesource-release-el7-1.noarch
 RUN yum clean all && rm -rf /var/cache/yum
 
-ENV GO_TARBALL https://dl.google.com/go/go1.12.12.linux-amd64.tar.gz
-RUN wget --progress=dot:giga ${GO_TARBALL} -O /tmp/golang.tar.gz
+ENV GO_VERSION 1.13.7
```

Comment:
I would keep either `GO_TARBALL` or that one, not both.

Reply:
Make sense, done.